airbyte-webapp/src/components/ui/Toast/Toast.module.scss

```diff
@@ -1,3 +1,89 @@
+@use "scss/colors";
+@use "scss/variables" as vars;
+
+$toast-icon-size: 13px;
+$toast-icon-container-size: 34px;
+
+@keyframes slide-up-animations {
+  0% {
+    transform: translate(-50%, -100%);
+    bottom: -49px;
```

Comment:
Since this value is reused a couple of times across this file that should be aligned, could we please extract this into a variable inside this file?

Reply:
replaced with another values 😅